docs/tutorials/Martini3/Small_Molecule_Parametrization_FastForward/index.qmd

```diff
@@ -0,0 +1,264 @@
+---
+title: "Parametrization of a new small molecule using Fast_Forward"
+format: html
+---
+
+<hr>
+In case of issues, please contact [Chris Brasnett](c.s.brasnett@rug.nl) or open an [issue on the repo](https://github.com/Martini-Force-Field-Initiative/Martini-Force-Field-Initiative.github.io/issues).
```

Comment:
```suggestion
Contributed by: <a href="https://github.com/csbrasnett">@csbrasnett</a>, <a href="https://github.com/ricalessandri">@ricalessandri</a>.

In case of issues, please contact [Chris Brasnett](c.s.brasnett@rug.nl) or open an [issue on the repo](https://github.com/Martini-Force-Field-Initiative/Martini-Force-Field-Initiative.github.io/issues).
```